include/aie/Dialect/XLLVM/IR/XLLVMAIE2IntrOps.td

```diff
@@ -351,4 +351,49 @@ def VectorMaxLtBf16IntrOp :
     Arguments<(ins VectorOfLengthAndType<[32], [BF16]>:$lhs,
                    VectorOfLengthAndType<[32], [BF16]>:$rhs)>;
 
+// ----- MIN ELEMENT -----
+
+def VectorMinGe8IntrOp :
+    AIEVec2_IntrOp<"vmin.ge8",
+        [TypeIs<"res",
+            LLVM_StructOf<[
+                VectorOfLengthAndType<[64], [I8]>,
+                I32]>
```

Comment:
If we have operands with 64 lanes, I would expect that the second result should be I64 instead of I32.

Reply:
Good catch. I will update both the min and max ops PRs.